app/views/admin/versions/_object_desc_and_link.html.haml

```diff
@@ -153,6 +158,9 @@
   = last_deleted_version.try(:title) || last_deleted_version.try(:name)
 
 - unless %w(Conference Subscription Registration User).include?(version.item_type)
-  = "in conference"
-  = link_to Conference.find(version.conference_id).short_title,
-          edit_admin_conference_path(id: Conference.find(version.conference_id).short_title)
+  - if conference
+    = "in conference"
+    = link_to conference_short_title,
+            edit_admin_conference_path(id: conference_short_title)
+  - else
+    in deleted conference
```

Comment:
We still have the ID the conference had though right?
We should show that...
`in deleted conference with ID 123`